prboom2/src/dsda/stretch.c

```diff
@@ -183,6 +183,15 @@ static void InitStretchParam(stretch_param_t* offsets, int stretch, enum patch_t
       break;
   }
 
+  // Raven HUD breaks in "patch_stretch_not_adjusted",
+  // so let's use "patch_stretch_doom_format" settings
+  if (raven && stretch == 0)
+  {
+    offsets->video = &video_stretch;
+    offsets->deltax1 = (SCREENWIDTH - WIDE_SCREENWIDTH) / 2;
+    offsets->deltax2 = (SCREENWIDTH - WIDE_SCREENWIDTH) / 2;
+  }
+
```

Comment:
Do like the code below. Put an `if` before the `switch` statement that sets `stretch`to `patch_stretch_doom_format` if necessary

Reply:
My thoughts process here is that this is a temporary fix, and by keeping this code isolated, it will be easier to cull once we figure out how to render the statusbar correctly in "not adjusted".

By putting an `if` in the `switch`, we are solidifying temp code that shouldn't be final.